ui/contexts/metametrics.js

```diff
@@ -125,10 +124,6 @@ export function MetaMetricsProvider({ children }) {
 
 MetaMetricsProvider.propTypes = { children: PropTypes.node };
 
-export function useMetaMetricsContext() {
-  return useContext(MetaMetricsContext);
-}
-
```

Comment:
`useMetaMetricsContext` might be useful for hooks

Reply:
I would prefer this over `useContext(MetaMetricsContext)`

Reply:
The problem is that we don't want the context to be consumed directly because the hooks for tracking events do additional work. I removed so as to make it more difficult to consume the context directly.

Reply:
I think we should either support it get rid of it altogether. Making it more difficult may not be so useful.